### ISSUE TITLE
Running the example results in an error

### ISSUE THREAD

**Issue by GMFranceschini** [StephenRicher/HiCFlow#1]
Dear developers, 
I am trying to test HiCFlow with your test data/genome. However, running the command line in the readme file (`snakemake --use-conda --conda-prefix ./envs/ --cores 4 --configfile example/config/config.yaml`), I encounter this error in early processing steps (fastqscreen).

It seems the genome is not correctly picked up - could you please advise?
Thank you

EDIT: 
I forgot to specify my system is Ubuntu 22.04.

```
Using fastq_screen v0.5.2
Option --subset set to 100000 reads
Reading configuration from '../config/fastq_screen.config'
Skipping DATABASE 'Human' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Human/Homo_sapiens.GRCh38'
Skipping DATABASE 'Mouse' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Mouse/Mus_musculus.GRCm38'
Skipping DATABASE 'Rat' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Rat/Rnor_6.0'
Skipping DATABASE 'Drosophila' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Drosophila/BDGP6'
Skipping DATABASE 'Yeast' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Yeast/Saccharomyces_cerevisiae.R64-1-1'
Skipping DATABASE 'Worm' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Worm/Caenorhabditis_elegans.WBcel235'
Skipping DATABASE 'Arabidopsis' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Arabidopsis/Arabidopsis_thaliana.TAIR10'
Skipping DATABASE 'Ecoli' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/E_coli/Ecoli'
Skipping DATABASE 'rRNA' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/rRNA/GRCm38_rRNA'
Skipping DATABASE 'MT' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Mitochondria/mitochondria'
Skipping DATABASE 'PhiX' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/PhiX/phi_plus_SNPs'
Skipping DATABASE 'Lambda' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Lambda/Lambda'
Skipping DATABASE 'Vectors' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Vectors/Vectors'
Skipping DATABASE 'Adapters' since no bowtie index was found at '/media/stephen/Elements/genomes/FastQ_Screen_Genomes/Adapters/Contaminants'
Using 'bowtie2' as bowtie2 path
No reference genomes were configured, please adjust configuration.
Traceback (most recent call last):
  File "/home/gianma/Desktop/HiCFlow/workflow/scripts/fastqScreen.py", line 55, in <module>
    sys.exit(function(**vars(args)))
  File "/home/gianma/Desktop/HiCFlow/workflow/scripts/fastqScreen.py", line 25, in runFastQScreen
    shutil.move(glob.glob(dirname + '/*_screen.txt')[0], dataOut)
IndexError: list index out of range

```

**Comment by StephenRicher** [comment]
Hi @GMFranceschini,

Apologies for the delay in getting back to you. I forgot to document on the README that fast_screen can't be run without first downloading the reference genomes. I have updated the README with instructions on how you can obtain these genomes or otherwise you can skip this step and simply comment out that part of the config.yaml. I've also made a couple of changes to the workflow so pull the most updated version of the main branch before attempting this.

Please do let me know if you run into any more problems.

Best wishes,
Stephen

**Comment by GMFranceschini** [comment]
Dear Stephen, thanks a lot for getting back to me. I installed FastqScreen, downloaded the genomes, and updated the paths in the `fastqscreen.config` file. I rerun with:

```
snakemake --use-conda --forceall --conda-prefix ./envs/ --cores 4 --configfile example/config/config.yaml
```

But fastqScreen is stuck again with a different error. From the log file:

```
usage: fastqScreen.py [-h] [--version] [--subset SUBSET] [--threads THREADS]
                      --plotOut PLOTOUT --dataOut DATAOUT
                      fastq config
fastqScreen.py: error: the following arguments are required: --plotOut
```

Perhaps I forgot to add some additional parameters somewhere?

**Comment by StephenRicher** [comment]
Hi @GMFranceschini, ah this was my oversight in a recent change. The --plotOut argument isn't used any more as the plots are all handled by multiQC but I forget to remove the parameter. If you pull the main branch to get the fix and then try again.

Thanks for your patience, hopefully it runs for you. I'm on a Mac at the moment but it should work on Ubuntu too.

**Comment by GMFranceschini** [comment]
Don't even mention that you have been very swift and helpful in your answers. In the meantime, I successfully ran the pipeline to completion without QC. Now I'll hopefully be able to get it to run on the cluster as well, Snakemake should handle that.

I also set up the files for human data + phasing and hopefully got them right. If you plan to expand the docs, a small section on how to get the required data might be helpful.

Also, I noticed that in your examples, there is an inconsistency between hg19 file names (haplotypes, GATKboundle) and hg38 file names (fasta and gtf), so you might want to check that as well. Closing the issue now, thank you!
